src/lib/components/helpers/popUpSizeHelper.ts

```diff
@@ -0,0 +1,14 @@
+export function setMinimumPopUpSize(
+    minimumHeight: number,
+    minimumWidth: number,
+    defaultHeight: number,
+    defaultWidth: number,
+): void {
+    if (window.outerHeight < minimumHeight && window.outerWidth < minimumWidth) {
+        self.resizeTo(defaultWidth, defaultHeight);
+    } else if (window.outerHeight < minimumHeight) {
+        self.resizeTo(window.outerWidth, defaultHeight);
+    } else if (window.outerWidth < minimumWidth) {
+        self.resizeTo(defaultWidth, window.outerHeight);
+    }
```

Comment:
This is still not a good way to express this

Consider doing something along these lines:

```
const width = logic to calculate what the final width should be
const height = logic to calculate what the final height should be
self.resizeTo(width, height);
```